src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfoNullable.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Text.Json.Serialization;
```

Comment:
Was `using` necessary?

Reply:
I thought it would be good to add an `ExtensionDataStatus` debug assert below (shouldn't be in nullable code-flow for extension data).